vllm/model_executor/layers/quantization/compressed_tensors/utils.py

```diff
@@ -106,8 +148,9 @@ def find_matched_target(layer_name: Optional[str], module: Module,
                                            True))
 
     if matched_target is None:
-        raise ValueError(f"Unable to find matching target for {module} in the "
-                         "compressed-tensors config.")
+        raise ValueError(
+            f"Unable to find matching target for {layer_name} in the "
```

Comment:
Made this change because printing the module in the error message sometimes leads to an AttributeError for fused modules like `QKV....`